README.md

```diff
@@ -17,6 +17,11 @@ DockerCon 2015 Hackathon Project
 
 UI at http://localhost:8008
 
+## Warning
+
+Please be aware, that the default configuration keeps only 2 images: `ubuntu:latest` and `busybox:latest`, so all other images
```

Comment:
Can you update the warning to this :

Sherdock is a Work in Progress an running sherdock **might lead to docker images being deleted** on the host. The default configuration keeps only 2 images: `ubuntu:latest` and `busybox:latest`, so all other images will be deleted. Run at your own risk.